k8s-install/scripts/install-cni.sh

```diff
@@ -77,6 +77,10 @@ do
 			continue
 		fi
 		cp $path $dir/
+		if [ "$?" != "0" ];
+		then
+			echo "Copy File Error, Please check whether disabled selinux on your host"	
```

Comment:
I think we should tweak the message so that it doesn't mention selinux, since there may be copy errors for other reasons.

For example:

```
echo "Failed to copy $path to $dir. This may be caused by selinux configuration on the host, or something else."
```

We probably also want to `exit 1` right after to indicate a failure.

Reply:
if use `exit 1`, kubernetes will constantly restart the pod , Is it ok ?

Reply:
Eventually it will hit `CrashLoopBackoff`.

My thinking was it's better to get notified of this than for it to fail silently - WDYT?

Reply:
OK， it`s better.